core/controllers/reader_test.py

```diff
@@ -159,6 +162,55 @@ def test_welcome_exploration(self):
             'Finnish', 'Yes! Oppia is the Finnish word for learn.',
             'What is the value of')
 
+    def test_soft_classification(self):
+        """Tests the string classifier exploration with responses that
+        trigger soft rules.
+        """
+        self.init_player(
+            '16', 'Testing String Classifier', 'do you think three things can')
```

Comment:
You're right. Since the string classifier is no longer a demo exploration I'm removing these tests. They are pretty much along the same pattern as ReaderClassifyTests and take up additional runtime.